pre-docs/inventory_version2.md

```diff
@@ -104,7 +109,7 @@ groups:
 
 Make sure you have [configured PuppetDB](./bolt_connect_puppetdb.md)
 
-##### Fact Lookups
+###### PuppetDB Fact Lookups
```

Comment:
This heading level is toooo deep; it won't show up properly on the docs site. Can we get rid of this heading entirely and make the discussion of PuppetDB fact lookups part of the section about the PuppetDB plugin?